crypto/serializer/build.info

```diff
@@ -1,2 +1,8 @@
-SOURCE[../../libcrypto]=serializer_meth.c serializer_lib.c serializer_pkey.c \
-        serializer_err.c
+SOURCE[../../libcrypto]=serdes_pass.c
```

Comment:
Not quite sure why you need separate SOURCE[../../libcrypto] here.

Reply:
I try to keep certain things separated...  maybe it's a bit over-zealous, but...

Reply:
It brings in questions (like Shane's) that seem unnecessary.  Split the lines if you feel strongly about it.